pmm/pmm2-release.groovy

```diff
@@ -36,8 +36,149 @@ pipeline {
+                                    fi
+                                    if [[ \${REPOSITORY} = PMM2-CLIENT ]]; then
+                                        REPOPATH=repo-copy/pmm2-client/yum
+                                    fi
+                                    cd /srv/UPLOAD/${PATH_TO_CLIENT}
+                                    #
+                                    # getting the list of RH systems
+                                    RHVERS=\$(ls -1 binary/redhat | grep -v 6)
+                                    #
+                                    # source processing
+                                    if [ -d source/redhat ]; then
+                                        SRCRPM=\$(find source/redhat -name '*.src.rpm')
+                                        for rhel in \${RHVERS}; do
+                                            mkdir -p /srv/\${REPOPATH}/release/\${rhel}/SRPMS
+                                            cp -v \${SRCRPM} /srv/\${REPOPATH}/release/\${rhel}/SRPMS
+                                            createrepo --update /srv/\${REPOPATH}/release/\${rhel}/SRPMS
+                                            if [ -f /srv/\${REPOPATH}/release/\${rhel}/SRPMS/repodata/repomd.xml.asc ]; then
+                                                rm -f /srv/\${REPOPATH}/release/\${rhel}/SRPMS/repodata/repomd.xml.asc
+                                            fi
+                                            gpg --detach-sign --armor --passphrase ${SIGN_PASSWORD} /srv/\${REPOPATH}/release/\${rhel}/SRPMS/repodata/repomd.xml
+                                        done
+                                    fi
+
+                                    # binary processing
+                                    pushd binary
+                                    for rhel in \${RHVERS}; do
+                                        mkdir -p /srv/\${REPOPATH}/release/\${rhel}/RPMS
+                                        for arch in $(ls -1 redhat/\${rhel}); do
+                                            mkdir -p /srv/\${REPOPATH}/release/\${rhel}/RPMS/\${arch}
+                                            cp -av redhat/\${rhel}/\${arch}/*.rpm /srv/\${REPOPATH}/release/\${rhel}/RPMS/\${arch}/
+                                            createrepo --update /srv/\${REPOPATH}/release/\${rhel}/RPMS/\${arch}/
+                                            if [ -f  /srv/\${REPOPATH}/release/\${rhel}/RPMS/\${arch}/repodata/repomd.xml.asc ]; then
+                                                rm -f  /srv/\${REPOPATH}/release/\${rhel}/RPMS/\${arch}/repodata/repomd.xml.asc
+                                            fi
+                                            gpg --detach-sign --armor --passphrase $SIGN_PASSWORD /srv/\${REPOPATH}/release/\${rhel}/RPMS/\${arch}/repodata/repomd.xml
+                                        done
+                                    done
+                                done
```

Comment:
please add `date +%s > /srv/repo-copy/version` as the last command before exiting ssh. it is going to be marker that we need to sync repos